site/get-started.markdown

```diff
@@ -0,0 +1,137 @@
+---
+title: Get Started
+page: get-started
+isGettingStarted: true
+---
+
+# Get started
+Welcome, new Haskeller! Read on to quickly set up your Haskell dev environment, execute your first lines of code, and get directions for further learning!
+
+## Content
+  - [Set up Haskell dev environment](#set-up-haskell-dev-environment)
+    - [GHCup: universal installer](#ghcup-universal-installer)
+    - [Editor](#editor)
+  - [Running first lines of code](#running-first-lines-of-code)
+  - [Writing your first Haskell program](#writing-your-first-haskell-program)
+  - [Join the community](#join-the-community)
+  - [Next steps](#next-steps)
+
+## Set up Haskell dev environment
```

Comment:
Under this header I'd add a little text like (with better phrasing as mine):

```
A complete haskell dev environment consist in the haskell tooling (compiler, language server client and build tool) and an editor compatible with the language server protocol. The quickest way to get all of those is:

- Use Ghcup to manage all haskell tooling
- Use vscode as your editor with the haskell extension.
```

I think it is important to anticipate how many things are involved, otherwise it isn't clear how much of the guide should I read. 

Let me know what you think

Reply:
Sure, sounds good, added it!